apps/web/src/modules/job/components/JobCard/components/JobCardFooter/index.test.tsx

```diff
@@ -20,25 +21,37 @@ describe('<Footer />', () => {
     'Money',
     'PersonOutlined',
   ])
+  const { mockUserType } = mockUser()
   jest.spyOn(window, 'alert').mockImplementation(() => {})
 
-  const { default: Footer } = require('./index') as typeof import('.')
+  const { default: JobCardFooter } = require('.') as typeof import('.')
 
   afterEach(() => {
     jest.clearAllMocks()
   })
 
   describe('normal behavior', () => {
     it('should render correctly', () => {
-      const { getByText } = render(<Footer {...footerProps} />)
+      mockUserType('ACTOR')
+      const { getByText } = render(
+        <JobCardFooter status="OPEN" {...footerProps} />,
+      )
```

Comment:
```suggestion
      mockUserType(UserType.ACTOR)
      const { getByText } = render(
        <JobCardFooter status={JobStatus.OPEN} {...footerProps} />,
      )
```
for all enum including others page